packages/kotti-ui/source/next/kotti-field-toggle/types.ts

```diff
@@ -10,9 +10,10 @@ export namespace KottiFieldToggle {
 
 export namespace KottiFieldToggleGroup {
 	export type Entry = {
-		disabled?: boolean
+		isDisabled?: boolean
```

Comment:
Breaking change, but still unused in yoda.

Reply:
@FlorianWendelborn please add to release notes regardless of course

Reply:
can u extend this to selects as well

we unify all options, supposedly

so the options on `KtFieldMultiSelect` and `KtFieldSingleSelect`

Reply:
https://github.com/3YOURMIND/kotti/releases/edit/untagged-1a4388e0038e68afaefc